renderer/components/Editor.tsx

```diff
@@ -260,12 +269,68 @@ export class Editor extends React.Component<Props, State> {
     }
   }
 
+  shift = (number: number, shift: number) => {
+    return number * Math.pow(2, shift);
+  }
```

Comment:
Let's rename the function to `bitShiftLeft` and rename the `number` argument to `value` just so it's not confusing to see `number: number`. We can also rename `shift` to `numPositions`.